ruby-training/trainees/alan/test_katas/01/math_maddness/spec/math_maddness_spec.rb

```diff
@@ -0,0 +1,50 @@
+require 'rspec/autorun'
+
+class Aritmetic_operations
+    
+    def self.add(first_number, second_number)
+       first_number +  second_number
+    end
+    
+    def self.substraction(first_number, second_number)
+        first_number - second_number
+    end
+
+    def self.multiply(first_number, second_number)
+        first_number * second_number
+    end
+
+    def self.divide(first_number, second_number)
+        first_number / second_number
+    end
+end
+describe Aritmetic_operations do
+    describe "#add" do
+        it "Sum the two numbers and the result is equal to 16" do
+             calculus = Aritmetic_operations.add(12, 4)
+            expect(calculus).to eq(16)
+        end
+    end
+
+    describe "#substraction" do
```

Comment:
subtraction* (please check all instances of this word in your code to correct spelling).